### PR TITLE
config: Add back simnet.

### LINKED ISSUE #418: simnet: Allow use.
Please can re revert 618cfc7cf177b2963a6a5f2691657a382d1ae453 ? We are using in in dcrdex and it does work.

### PR DESCRIPTION
**Pull request by JoeGruffins**
closes #418

### CODE REVIEW

**Review by jholdstock — CHANGES_REQUESTED** [review]
Apologies! I should have consulted. I had never heard anyone mention vspd on simnet and never seen any code which uses it. Could you share more about how/why you're using it?

**Comment by JoeGruffins** [comment]
Be in a bash enviroment, probably linux. Have tmux, dcrd, dcrwallet, and vspd in path. Get dcrdex master or better yet https://github.com/decred/dcrdex/pull/2482. go to /dex/testing/dcr and run ./harness. press ctrl + b then p to see the vspd running on simnet. We are using it to test ticket purchasing which is being added. It is working with the UI in that pr and looks quite nice.

**Comment by JoeGruffins** [comment]
adding to and fixing the params https://github.com/decred/vspd/compare/f8fa4e838d99f03e5b9bb9f4f557909f6eedf42e..b04d3d44616dd0b451004fbdb192bb0819a8dd0b

File: cmd/vspd/params.go [hunk 1]
```diff
@@ -49,3 +49,11 @@ var testNet3Params = netParams{
 func (n *netParams) dcp5Active(height int64) bool {
 	return height >= n.dcp0005Height
 }
+
+var simNetParams = netParams{
```
[the remark this answers is not in the document]
Comment (JoeGruffins) [reply cmd/vspd/params.go:47]: done!

File: cmd/vspd/params.go [hunk 2]
```diff
@@ -49,3 +49,11 @@ var testNet3Params = netParams{
...
+	Params:              chaincfg.SimNetParams(),
+	dcrdRPCServerPort:   "19556",
+	walletRPCServerPort: "19557",
+	blockExplorerURL:    "...",
+	minWallets:          1,
```
[the remark this answers is not in the document]
Comment (JoeGruffins) [reply cmd/vspd/params.go:52]: done!

**Review by davecgh — APPROVED** [review]

**Review by jholdstock — APPROVED** [review]

**Pull request merged**